pkg/util/cluster.go

```diff
@@ -233,3 +233,8 @@ func ClusterAccessCredentialChanged(newSpec, oldSpec clusterv1alpha1.ClusterSpec
 	}
 	return true
 }
+
+// IsClusterCompleteAPIEnablements tells whether the cluster's API enablements (.status.apiEnablements) is complete.
+func IsClusterCompleteAPIEnablements(clusterStatus *clusterv1alpha1.ClusterStatus) bool {
+	return meta.IsStatusConditionTrue(clusterStatus.Conditions, clusterv1alpha1.ClusterConditionCompleteAPIEnablements)
+}
```

Comment:
It's not necessary to have a one-line function.
But if you like, we can have a method at https://github.com/karmada-io/karmada/blob/master/pkg/apis/cluster/v1alpha1/cluster_helper.go.